src/blocks/ImageTiles/ImageTiles.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import Ublock from '@src/components/Ublock/Ublock'
+
+export interface ImageTilesProps {
+    uBlockFull: boolean
+    mainData: ImageTilesMainProps
+    subData: ImageTilesSubProps[]
+}
+
+export interface ImageTilesMainProps {
+    image: HTMLImageElement
+    header: string
+    paragraph: string
+}
+
+export interface ImageTilesSubProps {
+    image: HTMLImageElement
+    header: string
+    paragraph: string
+}
+
+const ImageTiles: React.FC<ImageTilesProps> = ({
+    uBlockFull,
+    mainData,
+    subData,
+}): JSX.Element => {
+
+    return (
+        <Ublock full={uBlockFull}>
+            <div className="b-imagetile">
+                <div className="imagetile__container">
+                    <a href="https://ravendesignsystem.github.io/rds/dev/blocks/main/imagetile/" className="imagetile__item">
+                        <img src={mainData.image.src} alt={mainData.image.alt} />
+                        <div className="imagetile__content">
+                            <h3>{mainData.header}</h3>
+                            <p>{mainData.paragraph}</p>
```

Comment:
Remember to create semantic data: This should be called description instead of paragraph

Reply:
Done

Reply:
👍